test/vesting-v1/vesting-v1.test.ts

```diff
@@ -453,32 +609,37 @@ describe("VestingV1", () => {
 
     it("fails if start time is too far in to the future", async () => {
       const timeInDistantFuture = BN(await getCurrentTimestamp())
-        .plus(MONTH * 3)
-        .plus(DAY);
+        .plus(MONTH_SECONDS * 3)
+        .plus(DAY_SECONDS);
       await expectRevert(async () => vesting.methods.activate(timeInDistantFuture).send({ from: projectWallet }), Error.StartTimeTooDistant);
     });
 
     it("fails if there isn't enough PROJECT_TOKEN allowance to cover total allocated", async () => {
       await setAmountForUser1();
       await expectRevert(async () => vesting.methods.activate(await getDefaultStartTime()).send({ from: projectWallet }), "ERC20: insufficient allowance");
+      await approveProjectTokenToVesting(TOKENS_PER_USER - 1);
+      await expectRevert(async () => vesting.methods.activate(await getDefaultStartTime()).send({ from: projectWallet }), "ERC20: insufficient allowance");
```

Comment:
Do we want to put `"ERC20: insufficient allowance"` and other repeated error strings into constants?